cocotb/share/makefiles/Makefile.inc

```diff
@@ -71,7 +71,7 @@ ifeq ($(filter $(PYTHON_ARCH),64bit 32bit),)
 endif
 
 # Set PYTHONHOME to properly populate sys.path in embedded python interpreter
-export PYTHONHOME := $(shell $(PYTHON_BIN) -c 'from sysconfig import get_config_var; print(get_config_var("prefix"))')
+export PYTHONHOME := $(shell $(PYTHON_BIN) -c 'import site; print(site.PREFIXES[0])')
```

Comment:
There is also `sys.prefix` https://docs.python.org/3/library/sys.html#sys.prefix

Not sure what is better.

Reply:
```suggestion
export PYTHONHOME := $(shell $(PYTHON_BIN) -c 'import sys; print(sys.prefix)')
```

Reply:
I prefer it, there is no question `sys.prefix` will be populated with exactly one path.